lib/ui/page/call/search/view.dart

```diff
@@ -326,3 +290,94 @@ class SearchView extends StatelessWidget {
     );
   }
 }
+
+/// [ReactiveTextField] styled as a search field.
+class _SearchField extends StatelessWidget {
+  const _SearchField(this.state, {this.onChanged});
+
+  /// State of the search [ReactiveTextField].
+  final TextFieldState state;
+
+  /// Callback, called when [_SearchField] changes.
+  final void Function()? onChanged;
+
+  @override
+  Widget build(BuildContext context) {
+    final style = Theme.of(context).style;
+
+    return ReactiveTextField(
+      key: const Key('SearchTextField'),
+      state: state,
+      hint: 'label_search'.l10n,
+      hintColor: style.colors.secondary,
+      maxLines: 1,
+      filled: true,
+      dense: false,
+      padding: const EdgeInsets.symmetric(vertical: 18),
+      style: style.fonts.normal.regular.onBackground,
+      onChanged: onChanged,
+      floatingLabelBehavior: FloatingLabelBehavior.always,
+      prefix: Padding(
+        padding: const EdgeInsets.only(left: 10, right: 10),
+        child: Transform.translate(
+          offset: Offset(0, 2),
+          child: SvgIcon(SvgIcons.search, width: 18, height: 18),
+        ),
+      ),
+      trailing: Obx(() {
+        final Widget child;
+
+        if (state.isEmpty.value) {
+          child = const SizedBox();
+        } else {
+          child = AnimatedButton(
+            key: const Key('ClearButton'),
+            onPressed: () => state.text = '',
+            child: SvgIcon(SvgIcons.clearSearch),
+          );
+        }
+
+        return SafeAnimatedSwitcher(duration: 200.milliseconds, child: child);
+      }),
+    );
+  }
+}
+
+/// [ChatTile] representing the provided [RxChat] as a recent [Chat] for [SearchView].
+class _ChatTile extends StatelessWidget {
+  const _ChatTile(
+    this.rxChat, {
+    super.key,
+    required this.onTap,
+    required this.selected,
```

Comment:
I suggest to make these parameters to have default values instead of marking those as `required`. `selected` can be `false` by default, and `onTap` may be `null` by default?